projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -624,7 +625,40 @@ function render_for_website( $data, $classes, $styles ) {
 			'success_message' => $data['success_message'],
 		)
 	);
-	?>
+
+	if ( Jetpack_Memberships::user_can_view_post() ) { ?>
+		<div <?php echo wp_kses_data( $data['wrapper_attributes'] ); ?>>
+		<div class="jetpack_subscription_widget">
+			<div class="wp-block-jetpack-subscriptions__container">
+				<div class="wp-block-jetpack-subscriptions__form-elements">
+					<p id="subscribe-submit"
```

Comment:
What do we need the `id` for?

Reply:
😅 it's used on [styles here](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/extensions/blocks/subscriptions/view.scss#L77), we definitely shouldn't be using id for this. Let's change that in a different PR?

Reply:
Yep sounds good!